angular/src/app/modules/delivery-management/delivery/request-resource-tab/request-resource-tab.component.ts

```
@@ -234,6 +242,15 @@ export class RequestResourceTabComponent
 
         this.editingRows[index][field] = false;
         abp.notify.success("Updated Status!");
+        const rs = await this.resourceRequestService.getResouceRequestCV(item.resourceRequestId).toPromise();
+        if (rs && rs.success) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)